moxin-frontend/src/app.rs

```diff
@@ -118,12 +118,6 @@ pub struct App {
     store: Store,
 }
 
-impl LiveHook for App {
-    fn after_new_from_doc(&mut self, _cx: &mut Cx) {
-        self.store = Store::new();
-    }
-}
-
```

Comment:
Removing the setup from `after_new_from_doc` because one unnecessary instance of the Store (and therefore a backend) were being created by deriving `Default` and being called by Makepad on initialization of the App.

Another solution would be to make Store and `Option` in `App`, but that introduces conditionals **everywhere**.

Instead I moved the creation to the custom `Default` implementation.